lnwallet/interface.go

```diff
@@ -152,6 +153,15 @@ type WalletController interface {
 	SendOutputs(outputs []*wire.TxOut,
 		feeRate SatPerVByte) (*chainhash.Hash, error)
 
+	// CreateSimpleTx creates a Bitcoin transaction paying to the specified
+	// outputs. The transaction is not broadcasted to the network. In the
+	// case the wallet has insufficient funds, or the outputs are
+	// non-standard, an error should be returned. This method also takes
+	// the target fee expressed in sat/vbyte that should be used when
+	// crafting the transaction.
+	CreateSimpleTx(outputs []*wire.TxOut,
```

Comment:
Nice addition! We may also want to follow up w/ an RPC similar to `signrawtranasction` (along with maybe exposing something like `listunspent`). This would allows users to craft custom transactions based on the available utxo set.